modules/phps/Cards/Goals/GoalTwoKeepers.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace Fluxx\Cards\Goals;
+
+use Fluxx\Game\Utils;
+
+class GoalTwoKeepers extends GoalCard
+{
+    public function __construct($cardId, $uniqueId)
+    {
+        parent::__construct($cardId, $uniqueId);
+        $this->keeper1 = -1;
+        $this->keeper2 = -1;
+    }
+
+    public function goalReachedByPlayer()
+    {
+        $winner_id = $this->checkTwoKeepersWin($this->keeper1, $this->keeper2);
+
+        return $winner_id;
+    }
+
+    function checkTwoKeepersWin(
+        $keeper_nbr_A,
+        $keeper_nbr_B,
+        $without_B = false
+    ) {
+        $cards = Utils::getGame()->cards;
+
+        $keeper_A = $cards->getCardsOfTypeInLocation(
```

Comment:
I think it's easier to think about this the other way around = get the card and check its position, rather than getting all the keepers card that matches this.

Reply:
agreed